lib/libutee/tee_api_operations.c

```diff
@@ -414,8 +414,8 @@ TEE_Result TEE_SetOperationKey2(TEE_OperationHandle operation,
 	operation->info.handleState &= ~TEE_HANDLE_FLAG_KEY_SET;
 
 	if (key1 != TEE_HANDLE_NULL) {
-		TEE_CopyObjectAttributes(operation->key1, key1);
-		TEE_CopyObjectAttributes(operation->key2, key2);
+		(void)TEE_CopyObjectAttributes1(operation->key1, key1);
```

Comment:
If `TEE_CopyObjectAttributes1()` returns `TEE_ERROR_CORRUPT_OBJECT` this function should return that code.